### PR TITLE
Refresh, inspired by Mautic Conference 2021

### PR DESCRIPTION
**Pull request by ekkeguembel**

### CODE REVIEW

**Comment by ekkeguembel** [comment]
Note that this includes a rephrasing of the mission statement as well as now officially mentions the NGO topic.

File: pages/08.community-team/chapter.md [hunk 1]
```diff
@@ -6,27 +6,34 @@ title: 'Community Team'
...
-* Grow the Community - global and local engagement
-* Increase active contribution
-* Facilitate communication and contribution
+* Support growth of the global as well as the local communities
+* Help the Mautic community be most ethical, accessible and inclusive
+* Provide really easy starting points for contribution
+* Organize and support Mautic events
+* Continue to improve community infrastructure
+* Connect with other Open Source communities
+* Work towards a Mautic NGO (e.g. „Mautic Association“)
```
Comment (RCheesley) [inline pages/08.community-team/chapter.md:19]:
```suggestion
```

I am not sure this is on the cards for some time, we said we would review whether there was a need for such an entity annually:

> Therefore, we propose to revisit this discussion in twelve months (September 2020) in the context of the Community Council to determine whether the community is at a point where an NGO would be of benefit.  At this point, if it is determined by the Community Council that an NGO should be established, Acquia will fully support any processes necessary to enable its creation.
_Source: https://www.mautic.org/blog/press/response-to-community-consultation-on-the-governance-model-proposal_

So on that basis I would not feel comfortable with the wording of 'working towards a Mautic NGO' seeing as we have not even established a need for one in the Community Council yet.

Open to your thoughts on this topic.

File: pages/08.community-team/chapter.md [hunk 2]
```diff
@@ -6,27 +6,34 @@ title: 'Community Team'
...
+* Define & refine official event formats
+* Help people to organize and promote local Mautic events
 
 But also
 
 * Understand community personas
 * Promote and help building a business on Mautic, and explain active contribution as an important element of it
 * Find, grow and support ambassadors
+* Connect & collaborate with other communities
+* Support funding
```
Comment (RCheesley) [inline pages/08.community-team/chapter.md:36]:
```suggestion
* Support fundraising initiatives
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
A couple of suggestions / points of clarification!

**Review by RCheesley — APPROVED** [review]

**Pull request merged**